pkg/subctl/operator/deploy/ensure.go

```diff
@@ -0,0 +1,103 @@
+package deploy
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	submariner "github.com/submariner-io/submariner-operator/pkg/apis/submariner/v1alpha1"
+	"github.com/submariner-io/submariner-operator/pkg/broker"
+	submarinerclientset "github.com/submariner-io/submariner-operator/pkg/client/clientset/versioned"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/datafile"
+
+	apiextension "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+func Ensure(config *rest.Config, remoteNamespace string, submarinerNamespace string,
+	clusterID string, serviceCIDR string, clusterCIDR string,
+	subctlData *datafile.SubctlData) error {
+
+	// Create the CRDs we need
+	apiext, err := apiextension.NewForConfig(config)
```

Comment:
nit: we can refactor the Endpoint/Cluster CRD creation later so it's shared with the broker

Reply:
Yes, the CRDs aren’t broker-specific so they need to be moved out and shared.

Reply:
And as you said, the will need to go into the operator itself eventually